src/ImageRunModal.jsx

```diff
@@ -744,7 +744,7 @@ export class ImageRunModal extends React.Component {
         );
 
         const defaultBody = (
-            <Form>
+            <Form onKeyDown={e => e.key !== 'Enter' || e.preventDefault()}>
```

Comment:
This was the kind of fix I was hoping would be possible when we talked about it on the call! Nice!

I'm curious why you did this as a negative check with `!` `||` instead of a positive check like this:

```
<Form onKeyDown={e => e.key === 'Enter' && e.preventDefault()}>
```

Reply:
Ah, I suppose old shell script writing habit (where `&&` has a strange semantics). But indeed it looks better with `&&`. I wanted to keep it as an expression (not a full statement) to keep the Arrow simple.

Reply:
There, changed to `&&`. Thanks for spotting!

Reply:
Yeah, it felt like a "double negative" to me, which is usually harder to read (both in written language and in code). It's true that shell scripting definitely has its issues. (I'm glad things like [shellcheck](https://www.shellcheck.net/) exist!)